runtime/Cpp/runtime/src/support/CPPUtils.h

```diff
@@ -21,14 +21,14 @@ namespace antlrcpp {
   // Using RAII + a lambda to implement a "finally" replacement.
   struct FinalAction {
     FinalAction(std::function<void ()> f) : _cleanUp { f } {}
-    FinalAction(FinalAction &&other) {
-      _cleanUp = other._cleanUp;
-      _enabled = other._enabled;
+    FinalAction(FinalAction &&other) :
+	_cleanUp(std::move(other._cleanUp)), _enabled(other._enabled)
+    {
```

Comment:
Coding style: opening brace on the end of the line opening the block (as it was originally).